src/UnwindRustSgx.c

```diff
@@ -0,0 +1,150 @@
+#define _GNU_SOURCE
+#include <link.h>
+
+#include <elf.h>
+#include <stdarg.h>
+#define max_log 256
```

Comment:
Don't put this define between some includes

Reply:
will fix.